docs/src/Scenario1/Evaluation_Scenario_1.md

```diff
@@ -46,53 +74,75 @@ function make_statified_model(pops; pop_assumption = (stratum, pop) -> 1)
     sys = structural_simplify(model)
     (C, sys)
 end
+```
+
+## Question 1
 
+> Start with a simple stratification with three age groups: young, middle-aged, and old.
+
+### Sub-question 1.a.
+> Begin with a situation where the population size across each age group is uniform: N_young = 2k, N_middle = 2k, N_old = 2k. Assume only one person in each age group is infectious at the beginning of the simulation. Let gamma = 1/14 days, and let R0 = 5. Assume gamma, beta, and R0 are the same for all age groups.
+
+> i. Simulate this model for the case where the 3x3 contact matrix is uniform (all values in matrix are 0.33)
+
+N.B.: Uniform `1/n_strata` is the default in our model creation function above.
+
+```@example scenario1
 (C, sys) = make_statified_model((2k, 2k, 2k))
 prob = ODEProblem(sys, [], (0, tf))
 sol = solve(prob)
-plot(sol, leg = :topright)
+plt_a1 = plot(sol, leg = :topright)
 ```
 
-## Question 1
-
-> Simulate this model for the case where there is significant in-group contact
+> ii. Simulate this model for the case where there is significant in-group contact
 > preference – you may choose the numbers in the matrix to represent this in-
 > group preference.
 
+We change the contact matrix by having significant (0.3)
```

Comment:
I don't get this: the resulting matrix doesn't have 0.3?

Reply:
I changed it after writing that to make the plots more interesting. I'll update the comment.